vllm/worker/hpu_model_runner.py

```diff
@@ -1794,9 +1978,11 @@ def mem_margin(self, value):
 
 
 def _maybe_wrap_in_hpu_graph(*args, **kwargs):
-    return htorch.hpu.wrap_in_hpu_graph(
-        HpuModelAdapter(*args, **kwargs), disable_tensor_cache=True
-    ) if htorch.utils.internal.is_lazy() else HpuModelAdapter(*args, **kwargs)
+    is_encoder_decoder_model = kwargs.get("is_encoder_decoder_model", False)
```

Comment:
can we define a `HPUModelAdapterEncoderDecoder` to handle changes in `HpuModelAdapter`?

Reply:
OK, I will follow this design.